test/data/PingFang/LICENSE.txt

```diff
@@ -0,0 +1,5 @@
+Copyright © 2015 DynaComware. All rights reserved.
```

Comment:
Hmm not sure if this is legal for us to distribute. May need to do what some of the other tests of macOS system fonts do: only run the test on macOS and point at the system installed version.

Reply:
@devongovett Font removed, test changed - is executed only if font is available.